File: bolareportsfrontend/src/app/@restmodels/header.request.ts
export class HeaderRequest {
    requestId!: string;
    affiliateCode!: string;
    requestToken!: string;
    sourceCode!: string;
    sourceChannelId!: string;
    requestType!: string;
    ipAddress!: string;
  lang: string;

    toString(): string {
        return '' +
            this.affiliateCode
            + this.ipAddress
            + this.requestId
            + this.requestToken
            + this.requestType
            + this.sourceChannelId
            + this.sourceCode;
    }

}

File: bolareportservices/src/java/com/ctrloption/bolareportservices/restmodels/commons/HeaderRequest.java
package com.ctrloption.bolareportservices.restmodels.commons;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class HeaderRequest {
    protected String sourceCode;
    protected String requestId;
    protected String requestToken;
    protected String requestType;
    protected String company;
    protected String ipAddress;
    protected String lang;
}
